linux/net.hhoney.tinycrate.yml

```diff
@@ -0,0 +1,29 @@
+id: net.hhoney.tinycrate
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.godotengine.godot.BaseApp
+base-version: '3.6'
+sdk: org.freedesktop.Sdk
+command: godot-runner
+finish-args:
+- --share=ipc
+- --socket=x11
+- --socket=pulseaudio
+- --device=all
+modules:
+- name: tinycrate
+  buildsystem: simple
+  sources:
+
+  - type: dir
+    path: ../
+
+  - type: file
+    url: https://github.com/cassidyjames/tiny_crate/releases/download/2024.09.18/Tiny-Crate.pck
```

Comment:
After merging the Desktop Entry and MetaInfo, we'll need a release and upload of a PCK file attached to it. Then we can update this to point to that PCK file (it's not super important at the moment because this manifest is just a test, and not the final one that gets submitted to Flathub).